tests/mocha/block_test.js

```diff
@@ -1335,12 +1341,139 @@ suite('Blocks', function() {
       }
     });
   });
-  suite('Icon Management', function() {
-    suite('Bubbles and Collapsing', function() {
+
+  suite('icon management', function() {
+    class MockIconA {
+      getType() {return 'A';}
+    }
+
+    class MockIconB {
+      getType() {return 'B';}
+    }
+    
+    suite.skip('adding icons', function() {
+      setup(function() {
+        workspaceTeardown.call(this, this.workspace);
```

Comment:
Why teardown at the beginning of the test?

Reply:
This tears down the old headless workspace and creates a new rendered workspace. Will add comment.